client/profile-wizard/index.js

```diff
@@ -278,11 +258,29 @@ class ProfileWizard extends Component {
 			} );
 	}
 
+	checkPluginsAreInstalled() {
+		const {
+			activePlugins,
+			profileItems,
+			updateProfileItems,
+			profileItemsLoaded,
+		} = this.props;
+
+		// Track plugins if already installed.
+		if (
+			profileItemsLoaded &&
+			profileItems.plugins !== 'installed' &&
+			activePlugins.includes( 'woocommerce-services' ) &&
+			activePlugins.includes( 'jetpack' )
+		) {
+			updateProfileItems( { plugins: 'installed' } );
```

Comment:
The goal here was just to make sure `plugins` gets set to something reasonable. Before it was trying to be set to `already-installed` but as mentioned that didn't work and wasn't very relevant anyway.

Reply:
Removed this based on discussion.